src/dcd/server/autocomplete/util.d

```diff
@@ -757,7 +757,23 @@ AutocompleteResponse.Completion makeSymbolCompletionInfo(const DSymbol* symbol,
 {
 	string definition;
 	if ((kind == CompletionKind.variableName || kind == CompletionKind.memberVariableName) && symbol.type)
-		definition = symbol.type.name ~ ' ' ~ symbol.name;
+	{
+		DSymbol* s = cast(DSymbol*)symbol;
```

Comment:
You don't need to cast away the const here, you could use `const(DSymbol)* s = symbol;` and if needed reassign it to s.type and thus keep the type safety